decentraland/helpers/index.ts

```diff
@@ -155,25 +256,31 @@ export const fundAccountWithMana = (accountAddress): ActionObject => {
 };
 
 // Note: This could live inside of the Action class as `buildLink()` function
-export const buildBlockchainUrlFromActionId = (actionId): string => {
+export const buildBlockchainUrlFromActionId = (actionId: any): string => {
   const networkName = getNetworkName();
   const transactionHash = actionId;
   const url = `https://${networkName}.etherscan.io/tx/${transactionHash}`;
   return url;
 };
 
+export const buildBlockchainUrlFromAddress = (address: string): string => {
+  const networkName = getNetworkName();
+  const url = `https://${networkName}.etherscan.io/address/${address}`;
+  return url;
+};
+
 // More about Toast component: https://docs.nativebase.io/Components.html#toast-def-headref
-const showToast = (msg): void =>
+const showToast = (msg: string): void =>
   Toast.show({ text: msg, duration: 3000, buttonText: "Okay" });
 
-export const showFatalError = (msg): void => console.error(msg);
-export const showError = (msg): void => showToast(`ERROR: ${msg}`);
-export const showWarn = (msg): void => showToast(`WARN: ${msg}`);
-export const showInfo = (msg): void => showToast(`${msg}`);
+export const showFatalError = (msg: string): void => console.error(msg);
+export const showError = (msg: string): void => showToast(`ERROR: ${msg}`);
+export const showWarn = (msg: any): void => showToast(`WARN: ${msg}`);
```

Comment:
I think here msg could be annotated as string
```suggestion
export const showWarn = (msg: string): void => showToast(`WARN: ${msg}`);
```